apps/src/StudioApp.js

```diff
@@ -2172,6 +2189,12 @@ StudioApp.prototype.handleHideSource_ = function(options) {
   }
 };
 
+StudioApp.prototype.handleIframeEmbedAppAndCode_ = function(options) {
```

Comment:
Also, what are all the things that enabling this config option does? This function could be a good place to explain that in a comment (even though some of the side effects happen outside this function).

Reply:
Removed the options parameter. Dave, im not entirely sure what your question is about - what is 'this config option' in the above context?

Reply:
I was referring to `config.level.iframeEmbedAppAndCode`, which causes this method to be called. apologies for the ambiguity.

Reply:
Ah! That makes sense. It would definetly be a good idea to document that somewhere, and the code itself definetly seems like a good place to put that documentation.

I'm not sure if its the /best/ place to put it, but it certainly seems like it'd do for now. I think in an ideal world, we'd have a seperate config file or something specifying all of the options we have for how a web app can be rendered, and that file would also contain the documentation for what the flags do. I don't think it'd be /too/ too complicated to introduce something like that into this codebase.

Reply:
I added documentation, but I put it in the route file - that seems to me like a good place because its the point of entry for this config option.

With the documentation, I tried not to be /too/ specific, because doing so can introduce tech debt if the documentation ever becomes out of date, but I specified at a high level what the route does and what files it affects.

Reply:
looks great, thanks for adding that!